uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequest.php

```diff
@@ -588,10 +595,11 @@ private static function validateLeaveDaysAgainstAbsenceTypeMaxConsecutiveLeaveDa
 
     $interval = $toDate->diff($fromDate);
     $intervalInDays = $interval->format("%a");
+    $maxConsecutiveLeaveDays = $absenceType->max_consecutive_leave_days;
 
-    if (!empty($absenceType->max_consecutive_leave_days) && $intervalInDays > $absenceType->max_consecutive_leave_days) {
+    if (!empty($maxConsecutiveLeaveDays) && $intervalInDays > $absenceType->max_consecutive_leave_days) {
```

Comment:
The `$absenceType->max_consecutive_leave_days` has already been made a variable, so why not use it here too?